src/Configuration/Levels/TypeCoverageLevel.php

```diff
@@ -95,7 +93,5 @@ final class TypeCoverageLevel
         AddParamTypeFromPropertyTypeRector::class,
         MergeDateTimePropertyTypeDeclarationRector::class,
         PropertyTypeFromStrictSetterGetterRector::class,
-        StrictArrayParamDimFetchRector::class,
-        StrictStringParamConcatRector::class,
```

Comment:
@TomasVotruba I can see it moved to "risky" level, I still prefer to just remove them, as while it usefull, param should cover from outside call/access (from method call or property fetch).